src/backend/distributed/commands/type.c

```diff
@@ -352,10 +352,14 @@ AlterTypeStmtObjectAddress(Node *node, bool missing_ok)
 	AlterTableStmt *stmt = castNode(AlterTableStmt, node);
 	Assert(AlterTableStmtObjType_compat(stmt) == OBJECT_TYPE);
 
+	ObjectAddress *address = palloc0(sizeof(ObjectAddress));
```

Comment:
same here: Does this change really belong to this PR? Maybe save these changes to other PR. And, in this PR mostly focus on the new GUC